stacks-signer/src/client/stacks_client.rs

```diff
@@ -167,6 +206,26 @@ impl StacksClient {
         todo!("Get the next possible nonce from the stacks node");
     }
 
+    /// Helper function to retrieve the account info from the stacks node for a specific address
+    fn get_account_entry(
+        &self,
+        address: &StacksAddress,
+    ) -> Result<AccountEntryResponse, ClientError> {
+        debug!("Getting account info...");
+        let send_request = || {
+            self.stacks_node_client
+                .get(self.accounts_path(address))
```

Comment:
You're passing `?proof=0` as the query string, right?  Otherwise, you're going to get a large amount of unneeded data back.

Reply:
No I did not, will update to do!